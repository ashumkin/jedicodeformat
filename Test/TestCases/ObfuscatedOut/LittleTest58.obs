unit littletest58;interface type tdummy=record case integer of 0:(erx:longword;word
:record filler:word;case integer of 0:(rx:word;byte:record rh:byte;rl:byte;end;);
end;);end;tsomeobject=class(TObject)private fdummy:array[0..8]of tdummy;public property
drxummy:tdummy read fdummy[0]write fdummy[0];property e:longword read fdummy[0].erx
write fdummy[0].erx;end;implementation end. 