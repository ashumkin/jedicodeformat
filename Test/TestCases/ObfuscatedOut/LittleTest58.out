unit littletest58;

interface

type
  tdummy = record
    case integer of
      0: (erx: longword;
        word: record
          filler: word;
          case integer of
            0: (rx: word;
            byte: record
              rh: byte;
              rl: byte;
            end;);
        end;);
  end;

  tsomeobject = class(TObject)
  private
    fdummy: array[0..8] of tdummy;
  public
    property drxummy: tdummy Read fdummy[0] Write fdummy[0];
    property e: longword Read fdummy[0].erx Write fdummy[0].erx;
  end;

implementation

end.
 